svgtoeagle.js

```diff
@@ -215,7 +216,7 @@ function drawSVG() {
   canvas.height = size.height*drawMultiplier;
   DRAWSCALE = drawMultiplier / SCALE;
 
-  if (EAGLE_FORMAT == "board") {
+  if (EAGLE_FORMAT == "board" && !LAYER_COLOR) {
     out("CHANGE layer "+EAGLE_LAYER+"; CHANGE rank 3; CHANGE pour solid; SET WIRE_BEND 2;\n");
   } if (EAGLE_FORMAT == "library") {
```

Comment:
Do we need `&& !LAYER_COLOR` here too?

Reply:
We need it here or we'll have duplicate 'Change layer' in the script